scripts/release/config.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
This is really beautiful. I love the structure of this whole thing.

Reply:
Thank you! 😁